src/shared/apis/queryCache.ts

```diff
@@ -178,6 +187,40 @@ export const resetQueryCache = (): void => {
   queryCache.resetCache()
 }
 
+const calculateHashedVariables = (allVars: Variable[], query: string) => {
```

Comment:
this logic is defined elsewhere. merge the two concepts so that we dont have competing definitions of a variable hash as this code is maintained